frontend/packages/dev-console/src/components/topology/__tests__/topology-test-data.ts

```diff
@@ -1449,6 +1450,96 @@ export const samplePipelineRun: FirehoseResult = {
   ],
 };
 
+export const serviceBindingRequest = {
+  data: {
+    apiVersion: 'apps.openshift.io/v1alpha1',
+    kind: 'ServiceBindingRequest',
+    metadata: {
+      name: 'analytics-deployment-D-wit-deployment-D',
+      namespace: 'testproject1',
+    },
+    spec: {
+      applicationSelector: {
+        group: 'apps',
+        resource: 'deployments',
+        resourceRef: 'analytics-deployment',
+        version: 'v1',
+      },
+      backingServiceSelector: {
+        group: '',
+        kind: undefined,
+        resourceRef: undefined,
+        version: undefined,
+      },
+      detectBindingResources: true,
+    },
+  },
+  model: {
+    abbr: 'SBR',
+    apiGroup: 'apps.openshift.io',
+    apiVersion: 'v1alpha1',
+    crd: true,
+    id: 'servicebindingrequest',
+    kind: 'ServiceBindingRequest',
+    label: 'ServiceBindingRequest',
+    labelPlural: 'ServiceBindingRequests',
+    namespaced: true,
+    plural: 'servicebindingrequests',
+  },
```

Comment:
Why do we need to add model here? we can import this model in the test spec file

Reply:
I mock the ```k8sCreate``` with data and model. But, yeah I will directly import the model in the spec file and test instead of adding it in data.

Reply:
Done.